pkg/operator/operator.go

```diff
@@ -122,32 +125,31 @@ func startCron() {
 
 func runCron() {
 	defer errors.Recover("cron failed")
```

Comment:
If a panic happens, will the error be sent to telemetry? Do we need to have a recover in the operator package, so we can send the error? If so, maybe we don't need `operator.Panic()` since otherwise the error would be reported twice. Same question applies to the recover middleware in the http server.